### ISSUE TITLE
There is an option Invite multiple Users to workspace, but it's not conveyed to the user.

### ISSUE THREAD

**Issue by Santhosh-Sellavel** [Expensify/App#4668]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___
Note: User should have a workspace, if not create one using the following steps
1. Click the FAB button 
2. Select New Workspaces to create a one.

## Action Performed:
1. Now, Go to Settings,
2. Go to your workspace
3. Select People 
4. Click Invite
5. Enter the email or phone option is shown.
6. There is no relevant message that notifies users that we can add multiple users entering emails comma separated.

## Expected Result:
There should be some help note that says add multiple users by adding email or phone comma separated

## Actual Result:
No such messages. 
<img width="387" alt="Screenshot 2021-08-15 at 1 44 44 AM" src="https://user-images.githubusercontent.com/85645967/129459003-b6b7d965-3f52-44be-a6d4-c4957b3be9d0.png">

## Workaround:
It's a state of confusion If the user is not aware then it's pain user will add emails one by one.
Like me I'm unaware for a while until got clear [here](https://expensify.slack.com/archives/C01GTK53T8Q/p1628970311367300)

## Platform:

Where is this issue occurring?

- Web
- iOS
- Android
- Desktop App
- Mobile Web

**Version Number:**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation
**Expensify/Expensify Issue URL:**

## Suggestion
`Email or Phone` place holder does not convey to users that multiple users can be invited. So,
1 . We should change the placeholder alone saying we can add emails or phone commas seperated.
2. We should change the placeholder and add a help text below email/phone that says we can add multiple users
Need help with help & placeholder messages from the team.

## Proposal
After discussion will update a technical solution

[View all open jobs on Upwork](https://www.upwork.com/ab/jobs/search/?q=Expensify%20React%20Native&sort=recency&user_location_match=2)

**Comment by MelvinBot** [comment]
Triggered auto assignment to @zanyrenney (`AutoAssignerTriage`), see https://stackoverflow.com/c/expensify/questions/4749 for more details.

**Comment by MelvinBot** [comment]
Triggered auto assignment to @johnmlee101 (`Engineering`), see https://stackoverflow.com/c/expensify/questions/4319 for more details.

**Comment by MelvinBot** [comment]
@johnmlee101 Whoops! This issue is 2 days overdue. Let's get this updated quick!

**Comment by MelvinBot** [comment]
@johnmlee101 Eep! 4 days overdue now. Issues have feelings too...

**Comment by marcaaron** [comment]
We were already discussing some other plans to address this issue so I'm going to close this in favor of https://github.com/Expensify/App/issues/4775. We could re-open this if there's no resolution or we decide this is something we want to do. Thanks!

**Comment by Santhosh-Sellavel** [comment]
@marcaaron 
Hey mate,

https://github.com/Expensify/App/issues/4775
It's pretty much the suggestion to how we can addressing this issue (Original issue).
We could have had the discussion to handle it differently. But why close this issue?

**Comment by marcaaron** [comment]
>It's pretty much the suggestion to how we can address this issue (Original issue).

Closed to avoid confusion and unnecessary work by doing a temporary change instead of the larger one we are already planning.

**Comment by Santhosh-Sellavel** [comment]
Thanks for clarifying, @marcaaron. Just one more thing, will I get credit for this issue? Since I pointed it earlier will I get a chance to work on  #4775  post design phase?

**Comment by marcaaron** [comment]
Contributors are eligible for a bonus if they report an issue, get their proposal to work on the issue accepted, and successfully merge a fix without regressions. So yes, feel free to try to ask for a bonus if you are assigned to work on this issue.

**Comment by marcaaron** [comment]
>Since I pointed it earlier will I get a chance to work on #4775 post design phase?

No, pointing out an issue does not guarantee that you will be selected to work on that issue.

**Comment by marcaaron** [comment]
https://github.com/Expensify/App/blob/main/CONTRIBUTING.md#proposing-a-job-that-expensify-hasnt-posted

**Comment by Santhosh-Sellavel** [comment]
Thanks @marcaaron understood! I'm aware that I should propose an acceptable solution to get the credit. Once the design phase is complete I'll submit a proposal there #4775

**Comment by Christinadobrzyn** [comment]
Hey @Santhosh-Sellavel  - following up on payment for reporting this issue. Just some background on why this issue was closed. Before you reported this, this fix was already being worked on, internally, between our eng team and design team. This is an example of why we’re now asking contributors to post a proposal in #expensify-open-source and wait for our eng team/QA team to review it to make sure there’s no overlap of other projects. 

Since this workflow wasn't fully detailed in August, our team has decided to make an exception and pay you for this proposal. The payment for the proposal will be sent 7 days after the pull request is deployed to production to allow for regression testing. I’ll send you a hire request in Upwork to pay you $250 for reporting this. 

Note, [this](https://github.com/Expensify/App/issues/4775) project is on hold at the moment.

**Comment by Santhosh-Sellavel** [comment]
@Christinadobrzyn Thanks for the update!

**Comment by Christinadobrzyn** [comment]
This job - https://github.com/Expensify/App/issues/4775 is done. Per the above conversation, I paid @Santhosh-Sellavel $250 for proposing this issue.
